packages/kbn-plugin-generator/sao_template/template/public/application.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { AppMountParameters, CoreStart } from '<%= relRoot %>/src/core/public';
+import { AppPluginDependencies } from './types';
+import { <%= upperCamelCaseName %>App } from './components/app';
+
+
+export const renderApp = (
+    { notifications, http }: CoreStart,
+    { navigation }: AppPluginDependencies,
+    { appBasePath, element }: AppMountParameters
+  ) => {
+    ReactDOM.render(
+      <<%= upperCamelCaseName %>App
+        basename={appBasePath}
+        notifications={notifications}
+        http={http}
+        navigation={navigation}
```

Comment:
Just adding a note that we might want to revise the structure of these props later, pending the discussions in #53029

Reply:
Will keep a close watch on this.